lib/block_view.go

```diff
@@ -4031,11 +4110,22 @@ func (bav *UtxoView) GetProfileEntryForUsername(nonLowercaseUsername []byte) *Pr
 	// defer to the db. If a mapping exists in the db, return it. If not, return
 	// nil.
 	// Note that the DB username lookup is case-insensitive.
-	dbProfileEntry := DBGetProfileEntryForUsername(bav.Handle, nonLowercaseUsername)
-	if dbProfileEntry != nil {
-		bav._setProfileEntryMappings(dbProfileEntry)
+	if bav.Postgres != nil {
+		profile := bav.Postgres.GetProfileForUsername(string(nonLowercaseUsername))
```

Comment:
Make sure that GetProfileForUsername does the same lowercasing as the original. It probably does, but just double check.